README.md

```diff
@@ -66,4 +66,24 @@ Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser
 
 ## Automated Tests
 
-Unit and integration tests can be run by running `pytest` in the console. Make sure you have installed dependencies first.
\ No newline at end of file
+Unit and integration tests can be run by running `pytest` in the console. Make sure you have installed dependencies first.
+
+
+## Running the app in a container
+
+For local development, first build the container by running:
+
+```
+docker build --target development --tag todo-app:dev .
```

Comment:
(I appreciate basically trivial currently, but it would be nice to also mention the production container too!)